arangod/Cluster/ClusterInfo.cpp

```diff
@@ -5844,6 +5844,32 @@ CollectionID ClusterInfo::getCollectionNameForShard(std::string_view shardId) {
   return StaticStrings::Empty;
 }
 
+auto ClusterInfo::getReplicatedLogsParticipants(std::string_view database) const
+    -> ResultT<
+        std::unordered_map<replication2::LogId, std::vector<std::string>>> {
+  READ_LOCKER(readLocker, _planProt.lock);
+
+  auto it = _newStuffByDatabase.find(database);
+  if (it == std::end(_newStuffByDatabase)) {
+    return {TRI_ERROR_ARANGO_DATABASE_NOT_FOUND};
+  }
+
+  auto replicatedLogs =
+      std::unordered_map<replication2::LogId, std::vector<std::string>>{};
+  for (auto const& [logId, logPlanSpecification] : it->second->replicatedLogs) {
+    std::vector<std::string> participants;
+    auto const& planParticipants =
+        logPlanSpecification->participantsConfig.participants;
+    participants.reserve(planParticipants.size());
+    for (auto const& pInfo : planParticipants) {
+      participants.push_back(pInfo.first);
+    }
+    replicatedLogs.emplace(logId, std::move(participants));
```

Comment:
maybe we can build it such that the leader is always listed first?

Reply:
Good ideea!